nf_core/modules.py

```diff
@@ -231,3 +234,81 @@ def has_valid_pipeline(self):
         if not os.path.exists(main_nf) and not os.path.exists(nf_config):
             log.error("Could not find a main.nf or nextfow.config file in: {}".format(self.pipeline_dir))
             return False
+
+
+class ModulesTestHelper(object):
+    def __init__(self, modules_dir=""):
+        self.modules_dir = modules_dir
+
+    # Add custom dumper class to prevent overwriting the global state
+    # This prevents yaml from changing the output order
+    # See https://stackoverflow.com/a/52621703/1497385
```

Comment:
Ah this old badger.. I've been vaguely wondering for a little while about switching to [`ruamel.yaml`](https://pypi.org/project/ruamel.yaml/) because of things like this. But have never quite been bothered enough to do it (and the main Python `yaml` module is pretty good most of the time).

Reply:
yes it does the job even though that's a bit ugly here 🤷